src/DetailsView/components/cards/collapsible-component-cards.tsx

```diff
@@ -31,6 +32,7 @@ export class CollapsibleComponentCards extends React.Component<CollapsibleCompon
 
     public render(): JSX.Element {
         const showContent = this.state.showContent;
+        const titleContainerProps = this.props.headingLevel ? { role: 'heading', 'aria-level': this.props.headingLevel } : undefined;
```

Comment:
We should make a few changes here:

1. Make heading level optional because that is how it behaves (specifying heading level 0 to get that behavior is odd).
2. Make this a strict null check (i.e. ```this.props.headingLevel != null```
3. Make the test case test for when we do not provide a heading level at all.

I think this would make for a more appropriate consumable component. My guess is that the intention of the original collapsible-container was similar.